core/frontend/src/DisplayStyleState.ts

```diff
@@ -50,7 +50,7 @@ export interface OsmBuildingDisplayOptions {
  * @extensions
  */
 export abstract class DisplayStyleState extends ElementState implements DisplayStyleProps {
-  /** @internal */
+  /** The name of the BIS class associated with this class. */
```

Comment:
i  don't think you need to copy this comment everywhere.

Reply:
Please provide a reason.
I don't like our documentation to include properties with zero explanation of what they do.
I don't like to assume people know that a `className` is obviously a BIS class name.
These are static properties so I don't think that the docs will produce a link to the documentation for the root type (EntityState) in which the property is initially defined.

Reply:
the "override" keyword means the documentation comes from the superclass.

Reply:
Resolve when satisfied.